prow/plugins/trigger/generic-comment.go

```diff
@@ -138,6 +138,30 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 	if pjutil.RetestRe.MatchString(gc.Body) || pjutil.RetestRequiredRe.MatchString(gc.Body) {
 		additionalLabels[kube.RetestLabel] = "true"
 	}
+	// run failed github actions
+	if pjutil.RetestRe.MatchString(gc.Body) || pjutil.TestAllRe.MatchString(gc.Body) {
+		headSHA, err := refGetter.HeadSHA()
+		if err != nil {
+			c.Logger.Warnf("headSHA unvailable, failed github actions for pr will not be triggered: %v", pr)
+		} else {
+			// pr.
+			failedRuns, err := c.GitHubClient.GetFailedActionRunsByHeadBranch(org, repo, pr.Head.Ref, headSHA)
```

Comment:
Since this will increase GitHub API requests, can we add a feature gate / option for this to be turned on in the trigger config?